packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -165,6 +165,17 @@ export function filterPackage(
 ) {
   if (builderSpec in localBuilders) return false;
   const parsed = npa(builderSpec);
+  const parsedVersion = semver.parse(parsed.rawSpec);
```

Comment:
Is there a scenario where semver.parse() will throw or does it always return `string|null`?

Reply:
I'm not 100% but I'm pretty sure that it does not throw.

Reply:
it throws, made a safe version
however keep in mind that it will throw anyway in the installation step, after trying for 3 times.